NuRadioMC/examples/08_RNO_G_trigger_simulation/simulate.py

```diff
@@ -93,15 +82,14 @@ def RNO_G_HighLow_Thresh(lgRate_per_hz):
 
 class mySimulation(simulation.simulation):
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, trigger_channel_noise_vrms=None, **kwargs):
         # this module is needed in super().__init__ to calculate the vrms
         self.rnogHarwareResponse = hardwareResponseIncorporator.hardwareResponseIncorporator()
-        self.rnogHarwareResponse.begin(trigger_channels=deep_trigger_channels)
+        self.rnogHarwareResponse.begin(trigger_channels=kwargs['trigger_channels'])
 
         super().__init__(*args, **kwargs)
-        self.logger = logging.getLogger("NuRadioMC.RNOG_trigger_simulation")
-        self.deep_trigger_channels = deep_trigger_channels
-
+        self.logger = logger
+        self.deep_trigger_channels = kwargs['trigger_channels']
```

Comment:
Does this actually get used? In l.199 it looks like these are hardcoded.

Reply:
I think I have another PR which does change this anyway